plugins/org.jkiss.dbeaver.ext.cubrid/plugin.xml

```diff
@@ -0,0 +1,233 @@
+                    <property name="zeroDateTimeBehavior" value="exception"/>
+                    <property name="logFile" value="cubrid_jdbc.log"/>
+                    <property name="logOnException" value="false"/>
+                    <property name="logSlowQueries" value="false"/>
+                    <property name="slowQueryThresholdMillis" value="60000"/>
+                    <property name="useLazyConnection" value="false"/>
+                    <property name="useSSL" value="false"/>
+                    <property name="clientCacheSize" value="1"/>
+                    <property name="usePreparedStmtCache" value="false"/>
+                    <property name="preparedStmtCacheSize" value="25"/>
+                    <property name="preparedStmtCacheSqlLimit" value="256"/>
+                    <property name="hold_cursor" value="true"/>
+                </driver>
+             </drivers>
+
+        </datasource>
+    </extension>
+    
+        <extension point="org.jkiss.dbeaver.dataTypeProvider">
+        <provider
+                class="org.jkiss.dbeaver.ext.cubrid.data.CubridValueHandlerProvider"
+                id="org.jkiss.dbeaver.ext.cubrid.data.CubridValueHandlerProvider"
+                label="Cubrid data types provider">
+            <datasource id="cubrid"/>
+            <!-- Date/time value handler - needed to provide native datetime value formatting. Formatting itself provided in drivers configuration. -->
+            <type kind="DATETIME"/>
+        </provider>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.objectManager">
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridTableManager"         objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridTable"/>
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridViewManager"          objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridView"/>
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridTableColumnManager"   objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridTableColumn"/>
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridPrimaryKeyManager"    objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridUniqueKey"/>
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridForeignKeyManager"    objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridTableForeignKey"/>
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridIndexManager"         objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridTableIndex"/>
+        <manager class="org.jkiss.dbeaver.ext.cubrid.edit.CubridProcedureManager"     objectType="org.jkiss.dbeaver.ext.cubrid.model.CubridProcedure"/>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.sqlDialect">
+        <dialect id="cubrid" parent="basic" class="org.jkiss.dbeaver.ext.cubrid.model.CubridSQLDialect" label="Cubrid" description="Dialect based on JDBC API information." icon="#database_icon_default" hidden="true">

```

Comment:
I don't know where '#database_icon_default' icon is used, but shouldn't I use 'cubrid icon.png'?
Other plugins use icons for the same purpose.